src/main/java/codeu/controller/ProfileServlet.java

```diff
@@ -0,0 +1,54 @@
+package codeu.controller;
+
+import codeu.model.data.User;
+import codeu.model.store.basic.UserStore;
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class ProfileServlet extends HttpServlet {
```

Comment:
Overall looks good, have a couple questions about the design, all you need to do is fix the build errors and do some testing with a skeleton jsp